### PR TITLE
MGDOBR-1330:Implement Create and Retrieve Bridge operations for v2 API Integration tests

### PR DESCRIPTION
**Pull request by ChughShilpa**
https://issues.redhat.com/browse/MGDOBR-1330 

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-XXX] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

**Comment by ChughShilpa** [comment]
Adding only test scenario for now as i can't test the execution locally

**Comment by ChughShilpa** [comment]
CRD operations implemented in this PR as `UPDATE` is not developed by engineers yet

File: integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/resources/BridgeResource.java
Comment (sutaakar) [inline integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/resources/BridgeResource.java:98]: Please revert this change, as it may be misleading. IMHO it is better to wait until the code is properly integrated.

File: integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/resources/features/bridge.feature
Comment (sutaakar) [inline integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/resources/features/bridge.feature:15]: IMHO this test can be commented out so far, as we cannot delete Bridge which is not in a final state.

File: integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/resources/BridgeResource.java
Comment (ChughShilpa) [reply integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/resources/BridgeResource.java]: Reverted

File: integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/resources/features/bridge.feature
Comment (ChughShilpa) [reply integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/resources/features/bridge.feature]: commented the delete scenario for now

**Review by sutaakar — APPROVED** [review]

File: integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/steps/Hooks.java
Comment (winklerm) [inline integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/steps/Hooks.java:63]: Does this mean that the integration tests will leave a Bridge instance(s) behind?

Comment (ChughShilpa) [reply integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/steps/Hooks.java:63]: @winklerm Yes, it will leave instances behind.
The cleanup procedure fails as the manager is not properly connected with operator (new APIs are still in development) and thus the scenario fails too.

Comment (winklerm) [reply integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/steps/Hooks.java:63]: Ok, thanks! I was just wondering if merging the code in this state could destabilize the environment, e.g. the dev cluster would be creating more and more Bridge instances that would never be deleted (and possibly some tests might fail on it)?

Bus since the PR is marked as do not merge, I guess it is fine now and we would just wait with merging until the clean up works. Wdyt?

**Review by winklerm — APPROVED** [review]
LGTM after the cleanup hook discussion is resolved (if the clean will not work in a near future and you need to merge this to get unblocked, I am fine with it, we just might need to keep an eye on the Bridge instances).

**Pull request merged**
